Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM ubuntu:14.04
```

Comment:
A previous commit had set the base image to `ubuntu:14:04` as a workaround for incorrectly packaged dependencies because ubuntu contains many of those dependencies (and much more besides). Proper use of `golang-builder` to package all of watchtower's dependencies into a single binary executable allows us to revert to a minimal base image.